seed/php-sdk/audiences/src/Foo/FooClient.php

```diff
@@ -25,18 +27,26 @@ public function __construct(
     }
 
     /**
-     * @param FindRequest request
+     * @param FindRequest $request
      * @param ?array{baseUrl?: string} $options
      * @returns mixed
      */
     public function find(FindRequest $request, ?array $options = null): mixed
     {
         $query = [];
-        if (request->optionalString != null) {
-            $query['optionalString'] = request->optionalString;
+        if ($request->optionalString != null) {
```

Comment:
i've wondering this in other languages; why not just construct this map all in one go and then filter out any property will null values after the fact?

Reply:
talked offline and made a `Nits` ticket to consider this improvement down the line